.circleci/test-server.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
```

Comment:
@nizar-m can these tests be run on local machines too?

Reply:
I have changed test-server.sh so that it can be run locally.